corehq/apps/accounting/models.py

```diff
@@ -392,7 +392,6 @@ class BillingAccount(ValidateModelMixin, models.Model):
     restrict_domain_creation = models.BooleanField(default=False)
     restrict_signup = models.BooleanField(default=False, db_index=True)
     restrict_signup_message = models.CharField(max_length=128, null=True, blank=True)
-    restrict_signup_email = models.CharField(max_length=128, null=True, blank=True)
```

Comment:
Generally when removing fields, there should be two PRs. One to remove references in code, the second to actually remove the field from the model. HQ was unavailable during deploy today because the old webworkers still referenced this field

Reply:
Ugh, sorry about that - pretty sure you explained this to me a few weeks back and I forgot.

Reply:
@emord @orangejenny I might need some explanation of this in case I ever need to do that ^^

Is it that it runs migrations prior to deploying the code?

Looking at one of the resulting sentry errors just shows the depths of the django ORM https://sentry.io/dimagi/commcarehq/issues/605080034/